# Equivalent to make maliput::api::RoadNetwork::road_geometry() and obtain the RoadGeometry information.
---
RoadGeometry road_geometry
